<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>470</width>
      <height>470</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="entryOne">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryTwo">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryThree">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryFour">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryFive">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entrySix">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entrySeven">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryEight">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryNine">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entryClear">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>325</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="solveTime">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>375</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>140</width>
      <height>210</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Keypad</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>entryOne</zorder>
   <zorder>entryTwo</zorder>
   <zorder>entryThree</zorder>
   <zorder>entryFour</zorder>
   <zorder>entryFive</zorder>
   <zorder>entrySix</zorder>
   <zorder>entrySeven</zorder>
   <zorder>entryEight</zorder>
   <zorder>entryNine</zorder>
   <zorder>entryClear</zorder>
   <zorder>solveButton</zorder>
   <zorder>resetButton</zorder>
   <zorder>solveTime</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
